src/Server/Server.php

```diff
@@ -168,8 +172,10 @@ public function getTokenCredentials(
     /**
      * Get user details by providing valid token credentials.
      *
-     * @throws CredentialsException If a "bad response" is received by the server
-     * @throws GuzzleException
+     * @param \League\OAuth1\Client\Credentials\TokenCredentials $tokenCredentials
+     * @param bool                                               $force
+     *
+     * @return \League\OAuth1\Client\Server\User
```

Comment:
These would be the examples of docblocks that have been added back which are completely redundant. The language already provides this return type. 

I don’t want to add any docblocks in that duplicate what the language provides. I went through and removed all the noise in a recent commit. 

Hope this makes sense.

Reply:
Agreed. As long as language provides its own typing, we shouldn't be adding redundant doc-blocks.